library/general/src/lib/config_file/config_file.rb

```diff
@@ -0,0 +1,29 @@
+module ConfigFile
+  class ConfigFile
```

Comment:
I do not like `GrubModel` as it sounds like model for whole grub, which is not true...so maybe `GrubConfModel`?